tests/xoto3/dynamodb/testing_utils.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from xoto3.dynamodb.put import PutItem
+from xoto3.dynamodb.types import TableResource, InputItem
+from xoto3.dynamodb.utils.table import extract_key_from_item
+
+
+def make_pytest_put_fixture_for_table(table: TableResource):
+    @pytest.fixture
```

Comment:
I would strongly recommend leveraging pytest's fixture system to avoid the need for defining closures or builders like this -- by defining the table itself as a fixture, put_item_fixture can be defined as a fixture that depends on the table fixture.

Reply:
hm, that's an interesting thought. i will give it a try and see how i feel. :)